Formula/gimp.rb

```diff
@@ -1,87 +0,0 @@
-class Gimp < Formula
```

Comment:
Thanks! 
As for this PR, it is not to be merged, it is just to see why do I have issues with debugging. So hopefully in the related issue we'll decide on what is finally to be deduplicated and I will do necessary migrations.